onnxruntime/contrib_ops/cpu/qlinear_global_average_pool.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include "core/util/math.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+class QLinearGlobalAveragePool final : public OpKernel {
+ public:
+  QLinearGlobalAveragePool(const OpKernelInfo& info) : OpKernel(info) {
+    int64_t nchw_layout;
+    ORT_ENFORCE(info.GetAttr("nchw", &nchw_layout).IsOK());
```

Comment:
For consistency, make this be a channels_last<int64_t> attribute like com.ms.QLinearConv and ReorderOutput.